### DISCUSSION TITLE
can this support use rest api to create a dag?

### DISCUSSION THREAD

**Discussion by tonglin96** [apache/airflow#20416] [Q&A]
### Description

can use rest api to create a dag or another way?

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!
